R/project.R

```diff
@@ -18,17 +18,24 @@
 #'
 #' }
 project <- function(default = NULL) {
-  renv_project(default = default)
+  renv_project_get(default = default)
 }
 
-renv_project_get <- function(default = NULL) {
-
+renv_project_get <- function(default = getwd()) {
   project <- Sys.getenv("RENV_PROJECT", unset = NA)
   if (is.na(project))
     return(default)
 
   project
+}
+
+renv_project_resolve <- function(project = NULL) {
+  project <- project %||% renv_project_get()
+  normalizePath(project, winslash = "/", mustWork = FALSE)
+}
 
+renv_project_is_active <- function(project = NULL) {
```

Comment:
This doesn't distinguish between "the user appears to have requested a particular project by setting the `RENV_PROJECT` environment variable" and "the project was actually loaded".